app/common/controllers/intercom.js

```diff
@@ -0,0 +1,28 @@
+module.exports = [
+    '$scope',
+    '$rootScope',
+    'UserEndpoint',
+function (
+    $scope,
+    $rootScope,
+    UserEndpoint
+) {
+    $rootScope.$on('event:authentication:login:succeeded', function () {
```

Comment:
So we can do anything that is here:
https://docs.intercom.io/configuring-intercom/the-intercom-javascript-api

Intercom does, usefully, record login/out automatically based on boot/shutdown that info will appear in the intercom interface so we get that upfront.